src/views/Orders.vue

```diff
@@ -18,28 +18,74 @@
 
     <ion-content>
       <div v-if="segmentSelected === 'open'">
-        <div v-for="order in orders" :key="order.orderId">
-          <OrderItemCard v-for="(shipGroup, index) in getShipGroups(order.items)" :key="index" :order="order" :shipGroup="shipGroup" @click.prevent="viewOrder(order)">
-            <template #packedTime>
-              <p></p>
-            </template>
-            <template #cardActionButton>
+        <div v-for="order in orders" :key="order.orderId" v-show="getShipGroups(order.items).length > 0">
```

Comment:
Is there a reason we can't add this condition directly onto the card? There is no scenario where we don't render card if there is an order to show.

Reply:
Sir, if we add this condition to the card then the div will take it space and thus result in empty spaces.

Reply:
We can get rid of the Div right?

Reply:
Sir, we need two loops, currently one is on div and another is on ion-card, so in any condition, we need to have a div for one loop.